src/pages/HomePage.tsx

```diff
@@ -122,23 +122,25 @@ const HomePage = () => {
               alt={intl.formatMessage({ id: "homepage-video-alt" })}
             />
           </Box>
-          <Text fontSize="md">
-            <FormattedMessage
-              id="homepage-how-it-works-paragraph"
-              values={{
-                appNameFull,
-                link: (children) => (
-                  <Link
-                    color="brand.600"
-                    textDecoration="underline"
-                    href={userGuideUrl()}
-                  >
-                    {children}
-                  </Link>
-                ),
-              }}
-            />
-          </Text>
+          {flags.websiteContent && (
```

Comment:
This loses us the whole of 

> Watch the video to get started. Learn more in the user guide.

... but not worth doing anything else at this point for the sake of < 24hrs.